localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -141,6 +141,15 @@ NDTScanMatcher::NDTScanMatcher()
   initial_pose_distance_tolerance_m_ =
     this->declare_parameter<double>("initial_pose_distance_tolerance_m");
 
+  use_cov_estimation_ = this->declare_parameter("use_covariance_estimation", use_cov_estimation_);
```

Comment:
Please do not use the default parameter and instead do as follow
```suggestion
  use_cov_estimation_ = this->declare_parameter<bool>("use_covariance_estimation");
```